recipes/k6/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set goname = "github.com/grafana/k6" %}
+{% set version = "1.0.0" %}
+
+{% set name = goname.split('/')[-1] %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://{{ goname }}/archive/v{{ version }}.tar.gz
+  sha256: 790e8a1d1171262095edbd5df5a74f18406d11ea88098560d0f18b7614c8b880
+
+build:
+  script:
+    - go build -trimpath -ldflags "-s -w" -o $PREFIX/bin/  # [unix]
+    - go build -trimpath -ldflags "-s -w" -o %LIBRARY_BIN%/  # [win]
+    - go-licenses save . --save_path="./license-files/" --ignore go.k6.io/k6 --ignore github.com/grafana --ignore buf.build/gen/go
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('go') }}
```

Please be explicit about the type of go compiler
```suggestion
    - {{ compiler('go-nocgo') }}
```